py-rse/project.Rmd

```diff
@@ -361,39 +318,33 @@ and recompile it as needed to produce executables:
     than interpreted programs,
     so something compiled on one computer might not work on another anyway.
 
-In practice,
-software developers handle all of this in different ways:
+Software developers handle this in different ways:
 
-1.  Put the source code for compiled languages in `src/`,
-    the runnable programs produced from this code in `bin/`,
-    and programs for interpreted languages in `scripts/`.
-    This makes version control easy---ignore what's in `bin/` and save everything else---but
+1.  Put the source code for compiled languages in `src`,
+    the runnable programs produced from this code in `bin`,
+    and programs for interpreted languages in `scripts`.
+    This makes version control easy---ignore what's in `bin` and save everything else---but
     means users have to look in two places to run things.
 
-2.  Put the source for compiled programs in `src/`
-    and both the compiler's output and interpreted programs in `bin/`.
+2.  Put the source for compiled programs in `src`
+    and both the compiler's output and interpreted programs in `bin`.
     This makes version control a little more complicated,
-    since some of what's in `bin/` needs to be saved and some doesn't,
+    since some of what's in `bin` needs to be saved and some doesn't,
     but means there's only one place to look for runnable programs.
-    (It's also easy to delete the hand-written scripts in `bin/` by accident
+    (It's also easy to delete the hand-written scripts in `bin` by accident
     when we only meant to delete the compiled programs,
     but since we have everything under version control,
     that's not a problem, right?)
 
-3.  Put all source files in `src/` and the compiler's output in `bin/`.
+3.  Put all source files in `src` and the compiler's output in `bin`.
     This is slightly simpler than the first option,
     but still means users have to look in two places for things they can run.
 
 As always,
 the approach matters less than being consistent
 and including a note in `CONTRIBUTING` or elsewhere to document the decision.
 
-In our Zipfs Law project,
-we don't have source files,
-so both shell (`*.sh`) and Python (`*.py`)
-files are included in `src/`.
-
-## How should I document the software in a project? {#project-software-docs}
+## Documenting Software {#project-software-docs}
```

Comment:
There's a link between this section and the "Documenting Packages" section in the packaging chapter (where we use Sphinx to build or Read the Docs site).

In `packaging.Rmd` we just put the bare minimum up on our Read the Docs site, but I think we should refer the reader back to this section for a discussion of all the things that could be included on that site (i.e. tutorials, FAQs, etc).